### DISCUSSION TITLE
Netbox 3.3.X compatibility

### DISCUSSION THREAD

**Discussion by juliouzzzz** [bb-Ricardo/netbox-sync#241] [Q&A]
Did someone try 1.3.0 with the last netbox major version ?
There are breaking changes in this major version, so i would know if the compatibility is ok

Thanks
Julien

**Comment by bb-Ricardo** [comment]
current 1.3.0 was tested with NetBox 3.3.2 for example. Version 1.3.0 contains changes to support NetBox version 3.3.X

**Reply by juliouzzzz** [reply]
Did you try with Major 3.4 new version ?
Thanks
Julien
